app/javascript/layouts/themes/growthtribe.css

```diff
@@ -49,34 +51,40 @@
 
   html {
     @apply theme-growthtribe;
+    font-family: Barlow, Helvetica, Arial, Verdana, Tahoma, sans-serif;
```

Comment:
https://github.com/growthtribeacademy/pupilfirst/blob/master/tailwind.config.js#L56 and
https://github.com/growthtribeacademy/pupilfirst/blob/master/app/javascript/layouts/themes/growthtribe.css#L35
is not enough ?

Reply:
haven't spotted it, since there was no font imported, but I agree, will fix ;)